src/utils/fileExports.js

```diff
@@ -190,8 +205,7 @@ export async function downloadCsvStatement(
     transactions,
     (tx) => tx.date < endDate,
   );
-  // TODO change to filter for txsInPeriod
-  const [txsInPeriod] = partition(txsBeforeEnd, (tx) => tx.date < startDate);
```

Comment:
this broke the whole fuctionality @mikozet but it is fixed in this PR again when merging.
(you basically removed all transactions, so none were included in #668)